athenet/sparsifying/derest/activation.py

```diff
@@ -0,0 +1,27 @@
+"""Functions that for any neuron and given range of inputs of layers, calculate
```

Comment:
Trolling ',' here.
"Functions that for any neuron and given range of inputs of layers calculate it's range of activation." sounds better
Same in other files